dspace-server-webapp/src/test/java/org/dspace/app/rest/CrossRefImportMetadataSourceServiceIT.java

```diff
@@ -0,0 +1,199 @@
+
+            String crossRefRespXmlResp = IOUtils.toString(crossRefResp, Charset.defaultCharset());
+
+            liveImportClientImpl.setHttpClient(httpClient);
+            CloseableHttpResponse response = mockResponse(crossRefRespXmlResp, 200, "OK");
+            when(httpClient.execute(ArgumentMatchers.any())).thenReturn(response);
+
+            context.restoreAuthSystemState();
+            ArrayList<ImportRecord> collection2match = getRecords();
+            collection2match.remove(1);
+            ImportRecord recordImported = crossRefServiceImpl.getRecord("10.26693/jmbs01.02.184");
+            assertNotNull(recordImported);
+            Collection<ImportRecord> recordsImported = Arrays.asList(recordImported);
+            matchRecords(new ArrayList<ImportRecord>(recordsImported), collection2match);
+        } finally {
+            liveImportClientImpl.setHttpClient(originalHttpClient);
+        }
+    }
+
+    @Test(expected = MethodNotFoundException.class)
+    public void crossRefImportMetadataFindMatchingRecordsTest() throws Exception {
+        context.turnOffAuthorisationSystem();
+        parentCommunity = CommunityBuilder.createCommunity(context)
+                                          .withName("Parent Community")
+                                          .build();
+
+        org.dspace.content.Collection col1 = CollectionBuilder.createCollection(context, parentCommunity)
+                                                              .withName("Collection 1")
+                                                              .build();
+
+        Item testItem = ItemBuilder.createItem(context, col1)
+                                   .withTitle("test item")
+                                   .withIssueDate("2021")
+                                   .build();
+
+        context.restoreAuthSystemState();
+        crossRefServiceImpl.findMatchingRecords(testItem);
+    }
+
+    private ArrayList<ImportRecord> getRecords() {
+        ArrayList<ImportRecord> records = new ArrayList<>();
+        //define first record
+        List<MetadatumDTO> metadatums  = new ArrayList<MetadatumDTO>();
+        MetadatumDTO title = createMetadatumDTO("dc", "title", null,
+                "State of Awareness of Freshers’ Groups Chortkiv State"
+                + " Medical College of Prevention of Iodine Deficiency Diseases");
+        MetadatumDTO author = createMetadatumDTO("dc", "contributor", "author", "L.V. Senyuk");
+        MetadatumDTO type = createMetadatumDTO("dc", "type", null, "journal-article");
+        MetadatumDTO date = createMetadatumDTO("dc", "date", "issued", "2016");
+        MetadatumDTO ispartof = createMetadatumDTO("dc", "relation", "ispartof",
+                                   "Ukraïnsʹkij žurnal medicini, bìologìï ta sportu");
+        MetadatumDTO doi = createMetadatumDTO("dc", "identifier", "doi", "10.26693/jmbs01.02.184");
+        MetadatumDTO issn = createMetadatumDTO("dc", "relation", "issn", "2415-3060");
```

Comment:
This isn't a valid field in DSpace. We don't have a `dc.relation.issn`.   I suspect you meant for this to be `dc.identifier.issn`, which **is a valid field**.